hack/make-rules/test-cmd-util.sh

```diff
@@ -3627,7 +3680,7 @@ run_stateful_set_tests() {
   # Pre-condition: no statefulset exists
   kube::test::get_object_assert statefulset "{{range.items}}{{$id_field}}:{{end}}" ''
   # Command: create statefulset
-  kubectl create -f hack/testdata/nginx-statefulset.yaml "${kube_flags[@]}"
+  kubectl create -f hack/testdata/rollingupdate-statefulset.yaml "${kube_flags[@]}"
```

Comment:
Just curious why changing this?
If `hack/testdata/nginx-statefulset.yaml` is no longer needed, please delete it.